README.adoc

```diff
@@ -80,6 +80,7 @@ In order to use wrappers for native SAT solvers (_e.g._, `MiniSatSolver`), you n
 When you use the solver (_e.g._ `JMiniSat`) first time, it loads the native library via `Loader.load("jminisat")` which tries to load them from 'resources' folder.
 
 * On Linux, place j-libs in `/path/to/kotlin-satlib/jni/src/main/resources/lib/linux64`.
+* On macOS, use `/path/to/kotlin-satlib/jni/src/main/resources/lib/osx64`.
```

Comment:
```suggestion
* On macOS, use `osx64` subfolder.
```

Reply:
Be consistent with the following line about Windows.
Or you may choose to merge them into one. But it is better to keep them separated.